src/gpodder/util.py

```diff
@@ -1397,12 +1398,22 @@ def format_time(value):
     '01:00:00'
     >>> format_time(10921)
     '03:02:01'
-    """
-    dt = datetime.datetime.utcfromtimestamp(value)
-    if dt.hour == 0:
-        return dt.strftime('%M:%S')
+    >>> format_time(86401)
+    '24:00:01'
+    """
+    hours = 0
+    minutes = 0
+    if seconds >= 3600:
+        hours = math.floor(seconds / 3600)
```

Comment:
You can leave out the `math` import if you use the `//` integer division operator here and below.

Reply:
Fixed, thank you!